src/main/java/javaserver/RequestParser.java

```diff
@@ -19,6 +20,21 @@ public static String getRequestURI(String rawRequest) {
 			return "";
 		}
 	}
+	
+	public static String getRequestData(String rawRequest) {
```

Comment:
Currently we are calling this method every time we want to create a request.  Maybe we can have a boolean method that checks if there is a body that needs to be parsed first.  Something like `hasBody` where we use a regular expression.